packages/core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@uma/core",
-  "version": "1.2.0",
+  "version": "1.2.2",
```

Comment:
h/t @mrice32 who pointed out that this version needs to be updated so that the import of `@uma/core^1.2.2` from `financial-templates-lib` resolves to the `core` directory and not from `financial-templates-lib/node_modules`

Reply:
This is merged into master, so you should be able to merge/rebase and this part of the diff should disappear.